src/redux/books/bookSlice.js

```diff
@@ -0,0 +1,21 @@
+import { createSlice } from '@reduxjs/toolkit';
+import books from '../../components/BookItem';
+
+const initialstate = {
+    books: books
+};
+
+const bookSlice = createSlice({
+    name: 'book',
+    initialState: initialstate,
+    reducers: {
+        removeItem: (state, action) => {
```

Comment:
- [ ] Good to have a removeItem/removeBook reducer :+1: But please make sure that you have `addBook` reducer for adding books to the redux store :pray: